### ISSUE TITLE
face orientation

### ISSUE THREAD

**Issue by aa755** [ageitgey/face_recognition#1033]
* face_recognition version: 1.2.3
* Python version:3.8.0
* Operating System: Manjaro Linux

### Description

Trying to detect my kid in an IP cam feed. 
The kid, often on the bed, can be in any orientation w.r.t. the camera. However, it seems that `face_detection`, even the cnn model, requires more-or-less vertical faces. `face_detection --model "cnn"` detects nothing in the following image, but it does detect the kid's face when I flip the image by 180 degrees

![t2](https://user-images.githubusercontent.com/499314/73130702-4bffb800-3fb2-11ea-8503-49036bbed3bb.jpg)

I can run the detection in parallel over all 45-degree rotations, but I hope there is a better way, especially because cnn is too slow.  The default model (not cnn) can't detect the kid even when the rotation is fixed. Perhaps it was not trained on kids?
The ideal solution should work even if the picture has different people in different orientations.
### What I Did

```
face_detection --model "cnn" dir/
```

**Comment by alessiosavi** [comment]
There is a parameter called `jitter` that is delegated to do data augmentation. Try to change the value to an higher number. This will increase the time spent during the face encoding but you have more probability to recognize the face

**Comment by aa755** [comment]
IIUC, messing with encodings will help if the face is being detected but misclassified as unknown or misclassified as another person. The kid's face doent get detected at all. Would changing jitter help with this?

**Comment by alessiosavi** [comment]
Of course, i don't know if the `jitter` parameter will rotate the photo of 180° during the data augmentation. But if the library is able to detect a face if you rotate the photo manually, then will be possible for tune the jitter parameter for achieve the success. In other case, you can try with `Keras` https://machinelearningmastery.com/how-to-configure-image-data-augmentation-when-training-deep-learning-neural-networks/
